### DISCUSSION TITLE
Adding Lux Power Inverter - what base template to use ?

### DISCUSSION THREAD

**Discussion by Daddypew** [springfall2008/batpred#963] [Q&A]
I'm loving the sound of the predbat features but have an unsupported Lux Power Tek LXP 5K Hybrid Inverter and 15KWh of GreenLinx batteries.
I have access to the necessary Inverter settings to read values and set charge/discharge etc (via LuxPython_DEV)
Is there a similar inverter template I can use as a base to try and configure for my system ?

Many thanks
Stuart

**Comment by nickjf20** [comment]
Hi, I've got this working read only with LXP-bridge. 
See: https://github.com/nickjf20/lxp-predbat/blob/main/app.yaml

**Reply by Daddypew** [reply]
Thanks for sharing, will have a look
